Formula/srecord.rb

```diff
@@ -3,6 +3,7 @@ class Srecord < Formula
   homepage "https://srecord.sourceforge.io/"
   url "https://downloads.sourceforge.net/project/srecord/srecord/1.64/srecord-1.64.tar.gz"
   sha256 "49a4418733c508c03ad79a29e95acec9a2fbc4c7306131d2a8f5ef32012e67e2"
+  license all_of: ["GPL-3.0-or-later", "LGPL-3.0-or-later"]
```

Comment:
Main `LICENSE` file is GPL 3.0 and listed on original tools, e.g. **srec_cat/main.cc**
```
// This program is free software; you can redistribute it and/or modify
// it under the terms of the GNU General Public License as published by
// the Free Software Foundation; either version 3 of the License, or (at
// your option) any later version.
```

Shared library was licensed with LGPL 3.0, e.g. **srecord/adler16.cc**
```
// This program is free software; you can redistribute it and/or modify
// it under the terms of the GNU Lesser General Public License as published by
// the Free Software Foundation; either version 3 of the License, or (at
// your option) any later version.
```